### ISSUE TITLE
Change index.html in generated backend

### ISSUE THREAD

**Issue by gastaldi** [fabric8-launcher/launcher-creator-backend#128]
It should contain a sentence, like `This is the backend, move along`

**Comment by gastaldi** [comment]
`Nothing to see here, move along`

**Comment by quintesse** [comment]
This would also mean removing the UIs some of the backend have, because some backend have them and others don't which is not very consistent.

**Comment by gastaldi** [comment]
@quintesse yes, let's remove all the UIs from the backend and have a single welcome page

**Comment by gastaldi** [comment]
@ALRubinger suggested using a more professional sentence, any suggestions?
